### ISSUE TITLE
String with semicolon doesn't persist

### ISSUE THREAD

**Issue by patrickmercurylabsio** [prazdevs/pinia-plugin-persistedstate#64]
I'm using `pinia-plugin-persistedstate` with Nuxt3 and can't seem to persist strings with semicolons in them. The code below illustrates the issue. When I remove the semicolon from the line `this.test = "foo; bar"` everything works as expected, however, when the semicolon is present the state doesn't seem to get persisted. Is this a bug or an error with my use of the package? Any assistance would be greatly appreciated.

```
import { defineStore } from "pinia";

export const useTestStore = defineStore("test", {
  state: () => {
    return {
      test: null,
    };
  },
  actions: {
    doTest() {
      console.log("PERSISTED", this.test);
      this.test = "foo; bar";
      console.log("FETCHED", this.test);
    },
  },
  persist: true,
});

```

**Comment by prazdevs** [comment]
Hey, thanks for pointing that out, it's actually an issue in the plugin for nuxt specifically, I have a fix for it, I'll push/release it ASAP

**Comment by prazdevs** [comment]
Alright, should be fixed with `@1.5.2` !

Let me know if you still encounter issues!

**Comment by patrickmercurylabsio** [comment]
Amazing, thanks!
